articles/introduction-to-numpy/index.md

```diff
@@ -0,0 +1,242 @@
+### A Not-So-Gentle Introduction to NumPy
```

Comment:
Some of the sentences are a little too long, which may result in the reader losing focus. It would be better if you could fix some of these by splitting it up into multiple sentences for better readability.

Example,
> When compared the performance of NumPy arrays vs Python lists as the number of elements increases is shown in the chart, with the breakeven size observed to be around 200 elements. 

Can be rephrased as,

**Let us compare Numpy arrays and Python lists. As shown in the chart, as the number of elements increases, the breakeven size is observed to be around 200 elements.**

For checking if an article is readable, please make use of this tool: [Hemingway App](http://www.hemingwayapp.com/)